target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-cap-ac.dts

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4018-mikrotik.dtsi"
```

Comment:
The idea of a shared DTSI is that it's shared. You don't share anything here.

Either create a DTSI that is actually shared between this device and the hap-ac2 (if that's possible), or put all code for your device into the DTS.

For a shared DTSI, the hard point would be to find a good name then, as it probably won't be shared between all ipq4018 Mikrotik devices, so just calling it "mikrotik" would probably not be a good idea (unless I'm wrong here).

Reply:
qcom-ipq4018-mikrotik.dtsi removed